robber/matchers/__init__.py

```diff
@@ -5,12 +5,12 @@
     'called_once',
     'called_once_with',
     'called_with',
-    'contain'
+    'contain',
     'equal',
-    'ever_called'
+    'ever_called',
```

Comment:
It's not like we have another class implemented, I just have to write 1-line code to register an equivalent chain. This case is the same as `greater_than`,  `above`, `more_than`. They're just there for user's convenience.